### PR TITLE
log yaml output to ocp.create() function

### PR DESCRIPTION
**Pull request by RazTamir**
This is to align with rook u/s testing

### CODE REVIEW

**Comment by RazTamir** [comment]
Verified:

15:52:49 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name csirbd-demo-pod
15:52:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /home/ratamir/ocs_automation/auth/kubeconfig create -f /tmp/POD_mkv7bxwy -o yaml
15:52:50 - MainThread - root - INFO - apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: '2019-07-07T12:52:50Z'
  name: csirbd-demo-pod
  namespace: default
  resourceVersion: '48616'
  selfLink: /api/v1/namespaces/default/pods/csirbd-demo-pod
  uid: 20b3ef50-a0b6-11e9-b7f0-0aa6ffd607c0
spec:
  containers:
  - image: nginx
    imagePullPolicy: Always
    name: web-server
    resources: {}
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /var/lib/www/html
      name: mypvc
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: default-token-t8cfh
      readOnly: true
  dnsPolicy: ClusterFirst
  enableServiceLinks: true
  imagePullSecrets:
  - name: default-dockercfg-qm5nd
  priority: 0
  restartPolicy: Always
  schedulerName: default-scheduler
  securityContext: {}
  serviceAccount: default
  serviceAccountName: default
  terminationGracePeriodSeconds: 30
  tolerations:
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  volumes:
  - name: mypvc
    persistentVolumeClaim:
      claimName: pvc-test-0715524241
  - name: default-token-t8cfh
    secret:
      defaultMode: 420
      secretName: default-token-t8cfh
status:
  phase: Pending
  qosClass: BestEffort

15:52:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n default --kubeconfig /home/ratamir/ocs_automation/auth/kubeconfig get Pod csirbd-demo-pod -o yaml

File: ocs_ci/ocs/ocp.py [hunk 1]
```diff
@@ -133,8 +133,9 @@ def create(self, yaml_file=None, resource_name='', out_yaml_format=True):
             command += f"{self.kind} {resource_name}"
         if out_yaml_format:
             command += " -o yaml"
-
-        return self.exec_oc_cmd(command)
+        output = self.exec_oc_cmd(command)
+        logging.info(f"{yaml.dump(output)}")
```
Comment (vasukulkarni) [inline ocs_ci/ocs/ocp.py]: can this be debug?

Comment (ebenahar) [reply ocs_ci/ocs/ocp.py]: +1
Printing the entire yaml will flood the log for info

Comment (RazTamir) [reply ocs_ci/ocs/ocp.py]: done

**Pull request merged**